bundles/framework/publisher2/tools/LayerSelectionTool.js

```diff
@@ -350,6 +350,7 @@ function() {
             }
         }
         if(me.__plugin) {
+            me.__plugin.teardownUI();
```

Comment:
Shouldn't it be the plugins responsibility to remove it's UI when it's stopped instead of the publisher functionality?